build/repo.targets

```diff
@@ -10,6 +10,14 @@
     </NPMPackage>
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(JAVA_HOME)' != ''">
+    <!-- Determine the path we need to search to find javac.exe -->
+    <JavacPath Condition="'$(OS)' == 'Windows_NT'">$(JAVA_HOME)\bin\javac.exe</JavacPath>
+    <JavacPath Condition="'$(OS)' != 'Windows_NT'">$(JAVA_HOME)/bin/javac</JavacPath>
+
+    <HaveJdk Condition="Exists('$(JavacPath)')">true</HaveJdk>
```

Comment:
You should also update the condition for the `RunJavaTests` target

Reply:
whoops!